firstvoices/backend/models/part_of_speech.py

```diff
@@ -34,6 +34,7 @@ class PartOfSpeech(BaseModel):
     class Meta:
         verbose_name = _("Part Of Speech")
         verbose_name_plural = _("Parts Of Speech")
+        ordering = ["title"]
```

Comment:
Any reason for adding all these ordering settings? They add a bit of overhead to every query

Reply:
I added these to resolve a couple of pagination warnings being shown when running pytests. These can be removed if they add a bit of overhead.